pkg/controller/iampolicy/iampolicy_controller.go

```diff
@@ -272,7 +291,31 @@ func checkUnNamespacedPolicies(plcToUpdateMap map[string]*policiesv1.IamPolicy)
 	return update, nil
 }
 
-func checkAllClusterLevel(clusterRoleBindingList *v1.ClusterRoleBindingList, clusterroleref string) (userV int) {
+func getGroupMembership(group string) ([]string, error) {
+	rv, err := (*KubeDynamicClient).Resource(openShiftGroupGVR).Get(context.TODO(), group, metav1.GetOptions{})
+	if err != nil {
+		// Ignore not found errors since it's possible this group isn't an OpenShift group
+		if errors.IsNotFound(err) {
+			log.Info(fmt.Sprintf("The group %s was not found. It may not exist or may not be an OpenShift group.", group))
+			return []string{}, nil
+		}
+
+		return nil, fmt.Errorf("failed to get the OpenShift group %s: %w", group, err)
+	}
+
+	users, found, err := unstructured.NestedStringSlice(rv.Object, "users")
+	if err != nil {
+		return nil, fmt.Errorf("the group was in an unexpected format: %w", err)
+	}
+
+	if !found {
+		return nil, fmt.Errorf("the group was in an unexpected format")
```

Comment:
Is it an error if it's not found? Or could we act as if the list was empty?

Reply:
I'm not sure. I figured this would be a case that we never hit since `users` is required in the `Group` kind.

What is your preference?

Reply:
Actually, I think you are right. I'd rather return an empty slice. I'll make the change.

Reply:
Yeah - we "shouldn't" hit this. `nil` is (in some respects) an empty list so it really really doesn't matter